doxygen/microservices.cpp

```diff
@@ -159,9 +159,6 @@
   - #msiNoTrashCan - Sets the policy to no trash can
   - #msiSetPublicUserOpr - Sets a list of operations that can be performed by the user "public"
   - #msiCheckHostAccessControl - Sets the access control policy
-  - #msiServerMonPerf - Monitors the servers' activity and performance
-  - #msiFlushMonStat - Flushes the servers' monitoring statistics
-  - #msiDigestMonStat - Calculates and stores a digest performance value for each connected resource
```

Comment:
Just noticed that the function prototypes for these still exist in reIn2p3SysRule.hpp.

They can be removed as well.

Reply:
ah, gotcha

Reply:
pushed, resolving.